samza-test/src/test/java/org/apache/samza/test/framework/StreamTaskIntegrationTest.java

```diff
@@ -263,4 +268,16 @@ public void genData(Map<Integer, List<KV>> inputPartitionData, Map<Integer, List
     }
   }
 
-}
\ No newline at end of file
+}
+
+class TestContext implements ExternalContext {
+  private int multiplier;
```

Comment:
Minor: please add final modifier for immutable field

Reply:
sure